eng/scripts/Get-PkgVersion.ps1

```diff
@@ -11,7 +11,7 @@ $repoRoot = Resolve-Path "$PSScriptRoot/../..";
 . (Join-Path ${repoRoot} eng common scripts logging.ps1)
 . (Join-Path ${repoRoot} eng scripts SdkVersion-Common.ps1)
 
-$versionFileLocation = Get-VersionHppLocaiton `
+$versionFileLocation = Get-VersionHppLocation `
```

Comment:
At first, I though, how could the package version scripts be working with this typo. Turns out it was consistently incorrect (including the function definition), so at least that was good :)